client/elements/suttaplex/sc-parallel-list.html

```diff
@@ -339,7 +339,7 @@
               }
               if (ids.includes('-')) {
                   const [id1, id2] = ids.split('-#');
-                  return `#${id1}--${id2}`;
```

Comment:
I think the double dash is needed for range selection in texts. @gradam can you confirm?

Reply:
I can change it back and remove them elsewhere.

Reply:
I've approved the ticket, but please verify that range selection is working. Otherwise, we can wait for @gradam's confirmation that this is ok.